File: 8kyu/(8 kyu) Abbreviate a Two Word Name/(8 kyu) Abbreviate a Two Word Name.cs
public class Kata
{
  public static string AbbrevName(string name)
  {
    string[] words = name.Split(' ');
    string res = "";
    res += words[0][0];
    res += '.';
    res += words[1][0];
    return res.ToUpper();
  }
}

File: 8kyu/(8 kyu) Abbreviate a Two Word Name/(8 kyu) Abbreviate a Two Word Name.go
package kata

import "strings"

func AbbrevName(name string) string{
  words := strings.Split(name, " ")
  res := ""
  res += string(words[0][0])
  res += "."
  res += string(words[1][0])
  return strings.ToUpper(res)
}
